ci-operator/step-registry/single-node/e2e-test2/test/single-node-e2e-test2-test-ref.yaml

```diff
@@ -0,0 +1,71 @@
+ref:
+  as: single-node-e2e-test2-test
+  from: tests
+  grace_period: 10m
+  commands: single-node-e2e-test2-test-commands.sh
+  timeout: 30800s
+  env:
+  - name: TEST_ARGS
+    default: ""
+    documentation: |-
+      Additional arguments to be passed to 'openshift-test'
+  - name: TEST_TYPE
+    default: "suite"
+    documentation: |
+      The type of test to perform.
```

Comment:
Same here, delete all these docs, just say that everything is the same as the original test step just duplicated due to CI limitations